internal/biz/model/common.go

```diff
@@ -1,315 +1,208 @@
 package model
```

Comment:
using this file to model shared constants in the domain model and also adding lightweight types, i.e Tiny Types that allow us to encapsulate validation within the type itself.